test/flow-types/regression-1114.js.flow

```diff
@@ -25,11 +25,6 @@ test.cb(t => {
 	t.end();
 });
 
-test.beforeEach(t => {
-	// $ExpectError: Unknown property "context"
```

Comment:
It's used elsewhere in this file, so might be best to keep it.

I think you have to be in the AppVeyor team to do that.